src/Symfony/Component/Validator/Constraints/CountryValidator.php

```diff
@@ -28,6 +29,10 @@ class CountryValidator extends ConstraintValidator
      */
     public function validate($value, Constraint $constraint)
     {
+        if (!class_exists('Intl')) {
+            throw new RuntimeException('symfony/intl is required to use the Country constraint');
```

Comment:
missing dot at end of message, also I'd suggest to be a little bit more explicit (same in other messages below):
`The "symfony/intl" component is required to use the Country constraint.`